src/background/responders.ts

```diff
@@ -135,28 +137,29 @@ export const responders: Responders<Action> = {
       ...updateFeedback(state, target, { isTweeting: false }),
     }
   },
-  clickDeleteScreenshot(state, { screenshotIndex }): Partial<StoreState> {
+  clickDeleteImage(state, { imageIndex }): Partial<StoreState> {
     return updateActiveFeedback(state, target => {
-      // The Screenshot array can't be empty
-      if (target.feedbackState.screenshots.length === 1) return {}
+      // The image array can't be empty
+      if (target.feedbackState.images.length === 1) return {}
 
-      // Removing the screenshot
+      // Removing the image
       // tslint:disable-next-line: readonly-array
-      const nextScreenshots = [...target.feedbackState.screenshots]
-      nextScreenshots.splice(screenshotIndex, 1)
-      return {
-        screenshots: nextScreenshots,
-      }
+      const nextImages = [...target.feedbackState.images]
+      nextImages.splice(imageIndex, 1)
+      return { images: nextImages }
     })
   },
-  startEditingScreenshot(state, { screenshotIndex }): Partial<StoreState> {
+  startEditingImage(state, { imageIndex }): Partial<StoreState> {
     return updateActiveFeedback(state, target => ({
-      editingScreenshot: {
-        screenshot: target.feedbackState.screenshots[screenshotIndex],
+      editingImage: {
+        image: target.feedbackState.images[imageIndex],
         color: '#fa759e',
       },
     }))
   },
+  imageUpload(state, { file }): Partial<StoreState> {
```

Comment:
Would we save the file or the file's data in the state? Why are we passing the file here?

Reply:
It's accessed by the listener, but not used here, so you're right that we can omit these from the function arguments of the responder.

Reply:
Fixed in latest checkin.